integr8ly/integreatly-operator
# remove user sso from multi tenant install type

# Issue link
https://issues.redhat.com/browse/MGDAPI-2752
remote user sso from multi tenant install type

Add quota specific to dev sandbox
https://issues.redhat.com/browse/MGDAPI-2754

Other change contains ns fix for OO related to dev sandbox. 

# Verification steps MGDAPI-2752
Install MT RHOAM and verify no usersso
Delete RHMI CR and ensure clean uninstall. 

# Verification steps MGDAPI-2754
Install MT RHOAM - ensure that the MT quota exists. Install with 100K quota
`oc get configmap quota-config-managed-api-service -n sandbox-rhoam-operator -o yaml`
Ensure backend_listenter has memory request 455 and memory limit 505
